test/functional/ui_job_wizard_controller_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_plugin_helper'
+
+class UiJobWizardControllerTest < ActionController::TestCase
+  def setup
+    FactoryBot.create(:job_template, :job_category=> 'cat1')
+    FactoryBot.create(:job_template, :job_category=> 'cat2')
+    FactoryBot.create(:job_template, :job_category=> 'cat2')
```

Comment:
Just add spaces before each of the rockets and it will be good to go, not sure why rubocop didn't complain about this

Reply:
Done!